bin/deploy-core

```diff
@@ -23,16 +23,24 @@ PILE_FAB=$(dapp create PileFab)
 SHELF_FAB=$(dapp create ShelfFab)
 COLLATERAL_FAB=$(dapp create CollateralFab)
 DESK_FAB=$(dapp create DeskFab)
+ADMIT_FAB=$(dapp create AdmitFab)
+ADMIN_FAB=$(dapp create AdminFab)
+
 
 # Deploy
-DEPLOYER=$(dapp create Deployer $GOD_ADDR $TITLE_FAB $LIGHTSWITCH_FAB $PILE_FAB $SHELF_FAB $COLLATERAL_FAB $DESK_FAB) 
+DEPLOYER=$(dapp create Deployer $GOD_ADDR $TITLE_FAB $LIGHTSWITCH_FAB $PILE_FAB $SHELF_FAB $COLLATERAL_FAB $DESK_FAB $ADMIT_FAB $ADMIN_FAB)
 $(seth send $DEPLOYER 'deployTitle(string memory, string memory)' "Tinlake Loan" "TLNT")
 $(seth send $DEPLOYER 'deployLightSwitch()')
 $(seth send $DEPLOYER 'deployCollateral()')
 $(seth send $DEPLOYER 'deployPile(address)' $CURRENCY)
 $(seth send $DEPLOYER 'deployShelf(address)' $APPRAISER)
 $(seth send $DEPLOYER 'deployValve()')
 $(seth send $DEPLOYER 'deployDesk()')
+$(seth send $DEPLOYER 'deployAdmit()')
+$(seth send $DEPLOYER 'deployAdmin(address)' $APPRAISER)
+
+$(seth send $APPRAISER 'rely(address)' $DEPLOYER)
```

Comment:
The admin contract should be allowed to use the appraiser.

I think it is better if the deployer adds permission instead of `GOD` with `seth`. 

What do you think?